success-criteria.md

```diff
@@ -113,34 +113,19 @@ behind this decision ought to be thoroughly documented.
 
 This document considers web advertising from three [points of
 view](https://www.w3.org/Consortium/Points/)[POINTS]: that of individual web
-users (both in aggregate and individually), publishers and their partners (both
+users (both in aggregate and individually), --publishers and their partners (both
 authors and the business model that funds them) and the delivery access
-mechanism (both connectivity and navigation).
-
-Any technology can be abused. Open societies do not attempt to suppress
-technology, but rather put [appropriate
-regulations](https://iabeurope.eu/all-news/iab-europes-press-statement-openrtb-and-eu-data-protection-law)
-[OPEN-RTB-EU-LAW] in place to define acceptable and unacceptable uses of that
-technology. For example, automobiles are not required to integrate functionality
-that technically prevents them from exceeding the speed limit. Instead, drivers
-are educated and trained in traffic rules, and drivers who violate speed limits
-are subject to fines and/or deprived of their permits. However, documenting
-specific criteria as to what constitutes a violation helps enable easier
-detection and reporting of non-compliance with the regulations that govern
-technology.
+mechanism (both connectivity and navigation).--
+
+Any technology can be abused. Any technology can be abused. Open societies consider all technologies but define ethical, appropriate, unethical and inappropriate uses and regulate or restrict accordingly. Documenting specific criteria as to what constitutes an ethical use of a technology helps enable easier detection and reporting of non-compliance with the regulations that govern societies and support the process of constructing such regulations.
 
 By documenting the defined norms and principles behind appropriate and
```

Comment:
> JR: I would delete this section. It is wrong to focus on one data protection principle, in this case accountability, in the absence of the others. In addition, it is not tenable to claim that accountability creates a requirement for “each participant that has access to data collection and processing to abide by its responsibility not to abuse the data under its control”. Data protection law does not say that data should be widely shared in order that recipients can decide whether to then protect it, and account for their actions. On the contrary, it says the opposite: data must not be exposed to unnecessary risk. This entire section should be removed. 

> [JWR] Article 24 of GDPR requires Controllers to “implement appropriate technical and organisational measures to demonstrate processing is performed in accordance with this regulation”. For this reason I think the paragraph is compatible with the GDPR. Could you advise if there is a better interpretation of Article 24?

I think my changes here around transparency and control lend a potential level of specificity to resolve this conflict, but it seems from this comment that it might make sense to make the ideas of "transparency" more clearly defined. I'm not sure who JR is? But would adding some more details here around transparency required for data handling under both a desire for accountability and a concern for regulation help with these questions?